ember-flight-icons/package.json

```diff
@@ -93,6 +93,7 @@
     "ember-source": "~3.26.1",
     "ember-source-channel-url": "^3.0.0",
     "ember-template-lint": "^3.5.1",
+    "ember-test-selectors": "^6.0.0",
```

Comment:
this may be :canofworms: so we can table if so, but in theory we'd want this behavior to ship with the addon so in consuming apps the `data-test-*` is stripped from production builds. The downside is that if the app isn't already doing that we would be introducing that behavior for them which _could?_ have unintended consequences? It definitely shouldn't though.

Reply:
HCP does have this addon [installed already](https://github.com/hashicorp/cloud-ui/blob/d2f33217d797b58a2c0f1106ec15d6cfc92a76d2/hcp/package.json#L117)